js/filteringBox.js

```diff
@@ -27,6 +27,78 @@ function openFilterModal() {
     closeFilterButton.classList.add("filter__box__topDiv__closeFilterBtn");
     closeFilterButton.innerHTML = "X";
     closeFilterButton.addEventListener("click", closeFilterModal);
+
+    // Checkbox divs
+    const checkDivMain = document.createElement("div");
+    filterBox.appendChild(checkDivMain);
+    checkDivMain.classList.add("filter__box__checkDivMain");
+
+    const checkDiv2 = document.createElement("div");
+    checkDivMain.appendChild(checkDiv2);
+    checkDiv2.classList.add("filter__box__checkDivMain__checkDiv2");
+
+    const checkDiv3 = document.createElement("div");
+    checkDivMain.appendChild(checkDiv3);
+    checkDiv3.classList.add("filter__box__checkDivMain__checkDiv3");
```

Comment:
You could add a function for repeating parts of the code, but since they don't repeat very often, the code is still very readable overall!

Reply:
Thank you for your feedback! I will absolutely consider adding a function instead when the code is more repetitive!

Reply:
Hi guys, you are doing an amazing work! I just want to add that Richard talked about seperation of concern a lot, I just want to suggest to Keep the structure in HTML. and focus JavaScript on behavior and interactivity. :) and of corse it is up to you to choose which approach you want to choose :)